tls/s2n_tls_parameters.h

```diff
@@ -58,6 +58,16 @@
 #define TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256  0xCC, 0xA9
 #define TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256      0xCC, 0xAA
 
+/* TLS Hybrid post-quantum definitions from https://tools.ietf.org/html/draft-campagna-tls-bike-sike-hybrid-00 */
+#define TLS_ECDHE_BIKE_RSA_WITH_AES_256_GCM_SHA3840 0xFF, 0x04
```

Comment:
Can we name these constants `TLS_HYBRID_*`?

Reply:
I am inclined to stick to the official names from https://tools.ietf.org/html/draft-campagna-tls-bike-sike-hybrid-00#section-6. Maybe we should update that and these constants?